src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,13 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+class LetterRemover private constructor() {
+    companion object {
+        @JvmStatic
+        fun removeLetter(string: String): CharArray {
```

Comment:
Please check this functionality
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html